amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/ShardSyncTask.java

```diff
@@ -73,6 +75,11 @@ public TaskResult call() {
             log.error("Caught exception while sync'ing Kinesis shards and leases", e);
             exception = e;
         } finally {
+            MetricsUtil.addSuccessAndLatency(scope,
```

Comment:
what happens to the latency metric in metric decorator?

Reply:
I didn't want to change the metric in the metric decorator as double counting counting time is due to the underlying Archiver metric factory